language/stdlib/modules/Libra.move

```diff
@@ -774,6 +778,7 @@ module Libra {
             !exists<CurrencyInfo<CoinType>>(Signer::address_of(lr_account)),
             Errors::already_published(ECURRENCY_INFO)
         );
+        assert(scaling_factor > 0, Errors::invalid_argument(ECURRENCY_INFO));
```

Comment:
I think you should have a new error reason here. E<RESOURCE_NAME> is typically used for existence or non-existence of a resource.

Reply:
@tzakian and I went over the other  places where  ECURRENCY_INFO is used in Libra.move and also the documentation for that error reason.  It is indeed used in many different ways so we concluded that my reuse of this reason is appropriate.  If you have a strong opinion, I can add a new reason.